tlsfuzzer/helpers.py

```diff
@@ -134,6 +134,31 @@ def sig_algs_to_ids(names):
     return ids
 
 
+def ext_names_to_ids(names):
+    """
+    Convert a string with names of extensions to list of IDs.
+
+    @type names: str
+    @param names: whitespace separated list of names of extension types.
+        Names can be specified either as full names (C{server_name}) or
+        as numerical IDs (C{0}).
+
+    @raises AttributeError: when the specified identifier is not defined
+        in ExtensionType
+    @return: list of int
+    """
+    ids = []
+    for name in names.split():
+        try:
+            num_id = int(name)
+        except ValueError:
+            num_id = getattr(ExtensionType, name)
```

Comment:
does getattr throw an error is name isn't found?

Reply:
Never mind. If you don't you will fail the self test in TestExtNamesToIds.

Reply:
yes, this  is the AttrubuteError that the doc string talks about